spec/serializers/api/beta/commodity_serializer_spec.rb

```diff
@@ -67,6 +67,8 @@
             search_references: '',
             validity_start_date: '1972-01-01T00:00:00Z',
             validity_end_date: nil,
+            end_line: nil,
+            declarable?: false,
```

Comment:
I think this should not always be false.
of course, there are declarable commodities ... or am I missing something? 🤔

Reply:
Yes it's not always false, here is the method that checks for declarable status <img width="727" alt="image" src="https://user-images.githubusercontent.com/108667911/184090626-009e7d1c-775c-42f3-8275-245394e51845.png">

And we've populated the test cases for the declarable status in the goods_nomenclature_spec.rb here
<img width="736" alt="image" src="https://user-images.githubusercontent.com/108667911/184091319-17440493-c1bb-4cdd-ae2c-bfa522ed365f.png">